header-footer-grid/templates/components/component-palette-switch.php

```diff
@@ -11,9 +11,10 @@
 
 use HFG\Core\Components\PaletteSwitch;
 
-$icon_type = component_setting( PaletteSwitch::TOGGLE_ICON_ID );
-$svg_icon  = PaletteSwitch::get_icon( $icon_type );
-$label     = component_setting( PaletteSwitch::PLACEHOLDER_ID );
+$icon_type   = component_setting( PaletteSwitch::TOGGLE_ICON_ID );
+$icon_custom = component_setting( PaletteSwitch::TOGGLE_CUSTOM_ID, '' );
+$svg_icon    = PaletteSwitch::get_icon( $icon_type, $icon_custom );
```

Comment:
We should make sure we escape the output of this variable when echoed, now that we're allowing custom code here. 🦭